engine/sync_manager.go

```diff
@@ -518,6 +530,7 @@ func (m *syncManager) worker() {
 						return
 					}
 
+					// TODO: Fix race issue to agent pointer.
 					c, err := m.get(exchangeName, enabledPairs[i], assetTypes[y])
 					if err != nil {
 						if err == errSyncPairNotFound {
```

Comment:
This no longer works and will need to be `errors.Is`. You can also remove `errSyncPairNotFound`